web/satellite/src/components/account/billing/BillingArea.vue

```diff
@@ -59,38 +61,49 @@
 		}
 	}
 
+	@media screen and (max-height: 900px) {
+		.billing-container__table-holder {
+			overflow-y: scroll;
+			height: 555px;
+
+			&::-webkit-scrollbar, &::-webkit-scrollbar-track, &::-webkit-scrollbar-thumb {
+				visibility: hidden;
+			}
+		}
+	}
+
 	@media screen and (max-height: 860px) {
-		.billing-overflow {
+		.billing-container__table-holder {
 			overflow-y: scroll;
-			height: 700px;
+			height: 535px;
 		}
 	}
 
 	@media screen and (max-height: 835px) {
-		.billing-overflow {
+		.billing-container__table-holder {
 			overflow-y: scroll;
-			height: 660px;
+			height: 495px;
 		}
 	}
 
 	@media screen and (max-height: 815px) {
-		.billing-overflow {
+		.billing-container__table-holder {
 			overflow-y: scroll;
-			height: 660px;
+			height: 495px;
 		}
 	}
 
 	@media screen and (max-height: 795px) {
-		.billing-overflow {
+		.billing-container__table-holder {
 			overflow-y: scroll;
-			height: 640px;
+			height: 475px;
 		}
 	}
 
 	@media screen and (max-height: 780px) {
```

Comment:
Shouldn't this block inherent the rules from line 96(https://github.com/storj/storj/pull/2127/files#diff-06d1d46adf3531fcd70bc6e4287c9c96R96)?

Reply:
Yes, sure. Done!